src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -203,7 +202,7 @@ def podp_download_and_extract_antismash_data(
             if output_path.exists():
                 Path.touch(output_path / 'completed', exist_ok=True)
 
-        except HTTPError:
+        except Exception:
```

Comment:
The only exception we should get should be HTTPError, because the rest should be handled by the code. The risk about using a generic `Exception` is that we are not aware of eventual other bugs there might be, that should be fixed differently and not using the try-except block. What do you think? This is my usual reasoning about this kind of logical block

Reply:
You're right. Here HTTPError and other errors are handled in the same way, so no need to differentiate them.

This commit is actually coupled with a [previous commit](https://github.com/NPLinker/nplinker/commit/d08851d6a72755a4960ebb17eaa0baf7709308a0) which makes sure deletion of extract_path can always happen.